libretro_core_options.h

```diff
@@ -428,7 +428,7 @@ struct retro_core_option_v2_definition option_defs_us[] = {
       BEETLE_OPT(gun_input_mode),
       "Gun Input Mode",
       NULL,
-      "Specify whether to use a mouse-controlled 'Light Gun' or a 'Touchscreen' input when device type is set to 'Guncon/G-Con 45' or 'Justifier'.",
+      "Choose whether to use a mouse-controlled 'Light Gun' or a 'Touchscreen' input when device type is set to 'Guncon/G-Con 45' or 'Justifier'.",
```

Comment:
"when the device type is set"...? "Touch Screen"? Also, one thing to keep in mind for the future: those controller types are unlocalizable at this moment.

Reply:
Both one word and two words variants of Touchscreen are acceptable (source: Merriam-Webster).
So, which one should we pick?

As for the controller types: where are those strings? RA or somewhere in this code?

Reply:
Oh, okay then with Touchscreen. Seems that the commonly used word in RA is also Touchscreen (with the exception of the 3DS, but that has a motive), so that issue can be ignored.

As for the types... I don't know. Those typically appear when going to the Controls section in the Quick Menu or the main Input menu, so... That could lead to something bigger.